css/tokens-default.css

```
@@ -126,7 +127,7 @@
   --color-text-on-base:         #ffffff;
   --color-text-on-neutral:      #ffffff;
   --color-text-on-dark:         #f3f4f6;
-  --color-backdrop:             rgba(0, 0, 0, 0.5);
+  --color-backdrop:             rgba(0, 0, 0, 0.5); /* --overlay-opacity not available in legacy browsers */
```

Stylelint (17.10.0)

[error] Expected "rgba" to be "rgb" (color-function-alias-notation)

(color-function-alias-notation)